fix(wallet): fees warning alignment fix

Nithishvb opened the pull request:
Closes #4674

This PR fixes the alignment issue with fees warning, ensuring a proper display on various devices and screen sizes. The previous alignment problem was affecting the user interface, and this fix aims to enhance the visual presentation of the fees warning.

kyranjamie commented:
Hi @Nithishvb thanks for the contribution. Would you mind rebasing your commits ? `git rebase origin/dev`?

Nithishvb commented:
Hi @kyranjamie completed the commit rebasing.

pete-watters commented on src/app/features/ledger/flows/bitcoin-tx-signing/ledger-bitcoin-sign-tx-container.tsx:
```diff
@@ -94,7 +95,11 @@ function LedgerSignBitcoinTxContainer() {
 
     try {
-      const btcTx = await signLedger(bitcoinApp, unsignedTransaction.toPSBT(), inputsToSign);
+      const btcTx = await signLedger(
```
Why are you making a change here?

pete-watters commented on src/app/components/fees-row/components/fees-row.layout.tsx:
```diff
@@ -23,7 +23,7 @@ export function FeesRowLayout(props: FeesRowLayoutProps) {
 
   return (
-    <HStack gap="space.04" width="100%" {...rest}>
+    <HStack gap="space.04" width="100%" {...rest} flexWrap={'wrap'} >
```
I checked and this pushes the warning down so it can be seen but it doesn't look too good:

![Screenshot 2023-12-14 at 06 38 41](https://github.com/leather-wallet/extension/assets/2938440/71989c21-b734-4feb-8f98-02d657265104)

I will ask for some help from our designers with this screen

pete-watters replied:
The design team said we can should this warning fit the full width available. If you can do that and update your PR as requested I can approve and merge it. Thank you for helping!

pete-watters commented:
Thanks for your contribution @Nithishvb, it's appreciated! 

I think for this PR you need to:
- rebase and squash your commit
- remove the un-needed commit to `ledger-bitcoin-sign-tx-container`
- await feedback from us on how it should look

Nithishvb commented:
Hi @pete-watters Thank you for your feedback! I've addressed the concerns raised in the pull request:

![Screenshot 2023-12-14 211910](https://github.com/leather-wallet/extension/assets/83941930/1051f4ef-4b50-4705-b7e0-74f06dd994ca)

pete-watters commented:
Thanks for this fix @Nithishvb . I still see this branch having 32 commits and it's also from a fork of our repo.

Can you please:
- checkout our repo 
- make a branch off `dev` with a single commit that makes changes to the one file you need to `src/app/components/fees-row/components/fees-row.layout.tsx` 
- commit that following our convention: `fix(wallet): fees warning alignment fix`

I would rebase this for you but as you have created a fork to work off it makes it more difficult to do that.

We ask for this as we want to keep a clean commit history

pete-watters reviewed (changes requested):
Thanks for this fix @Nithishvb . This fix is working well, thank you! Before I can merge I just need to clean up this PR a bit. 

I still see this branch having 32 commits.

Can you please:
- make a branch off `dev` with a single commit that makes changes to the one file you need to `src/app/components/fees-row/components/fees-row.layout.tsx` 
- commit that following our convention: `fix(wallet): fees warning alignment fix`


We ask for this as we want to keep a clean commit history

pete-watters commented:
Closing this as work done here https://github.com/leather-wallet/extension/pull/4710
